src/types/IstioObjects.ts

```diff
@@ -0,0 +1,82 @@
+import Namespace from './Namespace';
```

Comment:
The file IstioObjects.ts is confusing as it is a little bit ambiguous, in theory the pattern for naming was to use the elements close of the pages.
I understand the goal, it was to reuse some common types between workloads/services.
But well, I guess we can refactor in the future, I don't have a better name.
Once all main entities are consolidated perhaps we can pure refactor some interfaces and classes and clean something as I am sure we have some redundant classes under the types folder.
But no any action required, just leaving here a note to not forget.